Bottom: fd574e7d85e01a7897e4ccf9c7db402bcbc46d41
Top:    87a6fe7706e6779128f915f3fb4a6aa8a7d4b747
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 20:26:26 +0000

nelma: Fix prototype on set_layer() method to include the "..." flag.



---

diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index e7d0f9f..2180325 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -698,7 +698,7 @@ nelma_do_export(HID_Attr_Val * options)
 /* *** PNG export (slightly modified code from PNG export HID) ************* */
 
 static int 
-nelma_set_layer(const char *name, int group)
+nelma_set_layer(const char *name, int group, int empty)
 {
 	int             idx = (group >= 0 && group < max_layer) ?
 	PCB->LayerGroups.Entries[group][0] : group;
